io.edgehog.devicemanager.CellularConnectionStatus.json

```diff
@@ -0,0 +1,58 @@
+{
+  "interface_name": "io.edgehog.devicemanager.CellularConnectionStatus",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "datastream",
+  "ownership": "device",
+  "aggregation": "object",
+  "mappings": [
+    {
+      "endpoint": "/%{id}/carrier",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "Connectivity carrier operator name."
+    },
+    {
+      "endpoint": "/%{id}/cellId",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "The Cell ID in hexadecimal format, either 16 bit for 2G or 28 bit for 3G or 4G."
```

Comment:
I think there is no real value about using strings for cellId, also most of the APIs around are already using integers: https://developers.google.com/maps/documentation/geolocation/overview?hl=tr-tr. Let's change it to long integer.

Reply:
integer cannot work with 5G:
```
NR (5G) networks use the 36-bit New Radio Cell Identity (NCI) as is.
Valid range: 0–68719476735.
```
we must use long integer, which is int64.